op-guide/binary-deployment.md

```diff
@@ -46,7 +84,7 @@ Assume we have three machines with the following details:
 
 We run PD and TiKV on every node and TiDB on node1 only.
 
-**Note: This is just a demonstration here, please follow [Recommendations](./overview.md#deploying-recommendations) to deploy the cluster in production.**
+**Note: This is just a demonstration here, please follow [Recommendation](./recommendation.md) to deploy the cluster in production.**
```

Comment:
->**Note: This is just for demonstration, follow [Recommendation](./recommendation.md) to deploy the cluster in production.**